pkg/es/config/flags.go

```diff
@@ -1,8 +1,4 @@
-// Copyright (c) 2019 The Jaeger Authors.
-// Copyright (c) 2017 Uber Technologies, Inc.
-// SPDX-License-Identifier: Apache-2.0
-
-package es
+package config
```

Comment:
@yurishkuro thoughts on this change? I removed the `Options` and `namespaceConfig` structs entirely so we will be operating on the main Configuration struct now. As a result of this, I thought it made more sense to have the CLI flags live in `config`